content/index.md

```diff
@@ -1,18 +1,22 @@
 ---
 title: OSRA Infrastructure Project
```

Comment:
```suggestion
title: Open Robotics Infrastructure Project
```

This stands out to me since I prefer sentence case for titles but each of these words in the title is part of the proper noun "Open Robotics Infrastructure Project" 🪦ORIP?